cylc/flow/scheduler.py

```diff
@@ -843,9 +849,6 @@ def process_command_queue(self) -> None:
             try:
                 n_warnings: Optional[int] = self.get_command_method(name)(
                     *args, **kwargs)
-            except SchedulerStop:
-                LOG.info(f"Command succeeded: {cmdstr}")
-                raise
```

Comment:
This didn't seem possible, and indeed does not have any coverage hits

Reply:
:+1: I think this may have been legacy from how `cylc stop` was once implemented.